### PR TITLE
Bump distroless-iptables to v0.2.0

### PR DESCRIPTION
**Pull request by jingyuanliang**
Also, switched to registry.k8s.io and added make variables to allow specifying base image source and variation at build time.

/assign @MrHohn

### CODE REVIEW

File: scripts/Makefile [hunk 1]
```diff
@@ -30,7 +30,9 @@ ALL_PLATFORMS ?= linux/amd64 linux/arm linux/arm64 linux/ppc64le linux/s390x
 
 # The "FROM" part of the Dockerfile.  This should be a manifest-list which
 # supports all of the platforms listed in ALL_PLATFORMS.
-BASE_IMAGE ?= k8s.gcr.io/build-image/distroless-iptables:v0.1.2
+BASE_IMAGE_PREFIX ?= registry.k8s.io/build-image
```
Comment (aojea) [inline scripts/Makefile:33]: You have REGISTRY car below that seems to be the one that override prefix

Comment (aojea) [reply scripts/Makefile:33]: Or is only for release images?

**Comment by jingyuanliang** [comment]
/hold
I'm asking them if it's to be gke-release or gke-release-staging

File: scripts/Makefile (hunk 1 above)
Comment (aojea) [reply scripts/Makefile:33]: Nevermind, before coffee questions 😂

**Pull request closed without merging**